Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugLazMovement/DebugMovementParametersEditor.cs

```diff
@@ -0,0 +1,312 @@
+using UnityEngine;
+
+namespace Laz
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class DebugMovementParametersEditor : MonoBehaviour
```

Comment:
This looks very Clean.  Good Job :) 

Bar a bit of magic numbers here and there.